src/Components/About/About.test.js

```diff
@@ -21,17 +27,25 @@ describe('About', () => {
     const backEndEngineer2 = getByText(/dorion/i)
     const backEndEngineer3 = getByText(/sienna kopf/i)
 
-    expect(aboutHeading).toBeInTheDocument();
-    expect(appDescription1).toBeInTheDocument(); 
-    expect(appDescription2).toBeInTheDocument(); 
-    expect(appDescription3).toBeInTheDocument(); 
-    expect(frontEndHeading).toBeInTheDocument();
-    expect(frontEndEngineer1).toBeInTheDocument(); 
-    expect(frontEndEngineer2).toBeInTheDocument(); 
-    expect(frontEndEngineer3).toBeInTheDocument(); 
-    expect(backEndHeading).toBeInTheDocument();
-    expect(backEndEngineer1).toBeInTheDocument();
-    expect(backEndEngineer2).toBeInTheDocument();
-    expect(backEndEngineer3).toBeInTheDocument();
+    expect(aboutHeading).toBeInTheDocument()
+    expect(appDescription1).toBeInTheDocument() 
+    expect(appDescription2).toBeInTheDocument() 
+    expect(appDescription3).toBeInTheDocument() 
+    expect(frontEndHeading).toBeInTheDocument()
+    expect(frontEndEngineer1).toBeInTheDocument() 
+    expect(frontEndEngineer2).toBeInTheDocument() 
+    expect(frontEndEngineer3).toBeInTheDocument() 
+    expect(backEndHeading).toBeInTheDocument()
+    expect(backEndEngineer1).toBeInTheDocument()
+    expect(backEndEngineer2).toBeInTheDocument()
+    expect(backEndEngineer3).toBeInTheDocument()
+  })
+
+  it('should hide the Homecontainer when rendered', () => {
```

Comment:
Nice to get in a traditional React test for functions passed down as props - I missed how simple those are!